apps/messages/src/templates/welcome.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { MessageLayout, WelcomeMessage } from '@asap-hub/react-components';
+import { welcome } from '@asap-hub/message-templates';
+
+export default (
+  <MessageLayout
+    privacyPolicyHref={'https://hub.asap.science/privacy-policy'}
+    termsHref={'https://hub.asap.science/terms-and-conditions'}
```

Comment:
Feels like these should be constants somewhere?

Reply:
It be cool if the routing package could generate fully qualified urls given an environment

Reply:
Yeah idk if the routing package needs to do that since it's just `new URL(route({}).$, ORIGIN).href`. What I could indeed try to do here is re-use the `APP_ORIGIN` or whatever, I'll look into that. The `/privacy-policy` etc I mentioned briefly when doing the routing refactor, I didn't do that yet because the way that `Content.tsx` works in the FE to render them is really obscure, it's a refactor on my list :D

Reply:
done and rebased ^